web/src/components/alerts/fetcher.tsx

```diff
@@ -38,10 +39,34 @@ export const AlertFetcher: React.FC<AlertFetcherProps> = ({ children }) => {
       });
     return;
   }, []);
+
+  React.useEffect(() => {
+    getSilencedAlerts()
+      .then(result => {
+        setSilencedAlerts(
+          /* result.filter(rule => rule.status.state == 'active').map(rule => rule.id) */
+          result
+            .filter(rule => rule.status.state == 'active')
+            .map(rule => {
+              const matcher = rule.matchers.find(m => m.name == 'alertname');
+              return typeof matcher !== 'undefined' ? matcher.value : '';
+            })
+        );
+      })
+      .catch(() => {
+        console.log('Could not get alerts');
```

Comment:
nit:

```suggestion
        console.log('Could not get silenced alerts');
```
Should we also set silenced alertes to `[]` to show all alerts in that case ?

Reply:
Done, thanks